Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.9-slim-buster as base
+RUN apt-get update
+RUN apt-get install -y curl
+WORKDIR /app
+#RUN curl https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py
+RUN pip install poetry
+COPY poetry.toml .
+COPY pyproject.toml .
+#RUN poetry add gunicorn
```

Comment:
Let's remove these lines that you've commented out

Reply:
okay. But again this will be on the Module 6 commit